pages/phases/phone-assessment-interviews/reviewing-sme-assessments.md

```diff
@@ -16,6 +16,10 @@ Verify that every core competency has a rating of Does Not Meet, Meets, or Excee
 - Verify that the summary interview analysis is complete (with at least a few sentences).
 - Confirm the SME correctly addressed any initial issues you may have reported.
 
+### Calculating an Applicant's Final Assessment Rating
+
+After SMEs have addressed your feedback for their interview assessment forms, calculate the applicant's final assessment rating based on the SME's proficiency determination using the <a href="{{site.baseurl}}/toolkit/phone-assessment-interviews/interview-assessment-ratings/">Interview Assessment Rating Guide</a>.
+
```

Comment:
@bernars-usa there arent tiebreakers anymore in interviews. it was only there for HHS  because we still had 3s in TQ1. ties werent  possible at DOI.

Reply:
I see. Thanks. 👍